Tooll/Components/TextButtonEdit.xaml.cs

```diff
@@ -125,6 +132,14 @@ protected void TextEdit_LostFocus(object sender, RoutedEventArgs e)
             }
             TextEdit.Visibility = Visibility.Collapsed;
         }
+        
+        protected void TextEdit_PreviewMouseDown(object sender, MouseButtonEventArgs e)
+        {
+            if (e.ClickCount == 3)
+            {
+                XTextEdit.SelectAll();

```

Comment:
i tested it multiple times and saw no issues.
this was the shortest possible way of doing it, and i copied the style of handling this kind of event from the [MainWindow.xaml.cs](https://github.com/framefield/tooll/blob/master/Tooll/MainWindow.xaml.cs)
```C#
        private void OnDragMoveWindow(object sender, MouseButtonEventArgs e)
        {
            if (e.ClickCount == 2)
            {
                if (WindowState == WindowState.Maximized)
                {
                    WindowState = WindowState.Normal;
                }
                else
                {
                    WindowState = WindowState.Maximized;
                }                
            }
            else
            {
                DragMove();
            }                 
        }
```